layout/explore-detail/explore-detail-actions.js

```diff
@@ -3,7 +3,10 @@ import { createAction, createThunkAction } from 'redux-tools';
 import WRISerializer from 'wri-json-api-serializer';
```

Comment:
In this same file, you haven't changed this https://github.com/resource-watch/resource-watch/blob/b7b8c45613e8215ca2907cb99d663a9fc88bb6bd/layout/explore-detail/explore-detail-actions.js#L24, any reason to not do it? Or just forgot about it?

Reply:
fixed here: 
17c2178